digiroad2-oracle/src/main/scala/fi/liikennevirasto/digiroad2/service/pointasset/masstransitstop/TerminatedBusStopStrategy.scala

```diff
@@ -51,4 +51,5 @@ class TerminatedBusStopStrategy(typeId: Int, massTransitStopDao: MassTransitStop
     (resultAsset, PublishInfo(Some(resultAsset)))
   }
 
+  override def isFloating(persistedAsset: PersistedMassTransitStop, roadLinkOption: Option[RoadLinkLike]): (Boolean, Option[FloatingReason]) = { (true, Some(FloatingReason.TerminatedRoad)) }
```

Comment:
The isFloating from TierekisteriBusStopStrategy is doing something similar, if not the same, to this that I think that method is not doing anything, only returning (false, None).